TicTacToe/first.js

```diff
@@ -0,0 +1,116 @@
+let count = 1;
+let score1 = 0;
+let score2 = 0;
+let draw = 0;
+const boardArray = document.querySelectorAll(".restart");
+const play = document.getElementById("play");
```

Comment:
```suggestion
const playBtn = document.getElementById("play");
```
Acho que faz sentido dizer o tipo de elemento q é. Neste caso o botão de dar play